src/main/java/com/cupid/jikting/like/service/LikeService.java

```diff
@@ -19,17 +19,22 @@ public class LikeService {
 
     private final TeamMemberRepository teamMemberRepository;
 
-    @Transactional(readOnly=true)
+    @Transactional(readOnly = true)
     public List<LikeResponse> getAllReceivedLike(Long memberProfileId) {
         return getTeamMemberById(memberProfileId)
                 .getReceivedTeamLikes()
                 .stream()
-                .map(LikeResponse::from)
+                .map(LikeResponse::fromReceivedTeamLike)
                 .collect(Collectors.toList());
     }
 
-    public List<LikeResponse> getAllSentLike() {
-        return null;
+    @Transactional(readOnly = true)
+    public List<LikeResponse> getAllSentLike(Long memberProfileId) {
+        return getTeamMemberById(memberProfileId)
+                .getSentTeamLikes()
+                .stream()
+                .map(LikeResponse::fromSentTeamLike)
+                .collect(Collectors.toList());
```

Comment:
p1. 반복되는 코드는 아래처럼 재사용하면 좋을 것 같아요!

```suggestion
    public List<LikeResponse> getAllReceivedLike(Long memberProfileId) {
        return toLikeResponse(getTeamMemberById(memberProfileId).getReceivedTeamLikes());
    }
     
    public List<LikeResponse> getAllSentLike(Long memberProfileId) {
        return toLikeResponse(getTeamMemberById(memberProfileId).getSentTeamLikes());
    }
     
    public List<LikeResponse> toLikeResponse(List<TeamLike> teamLikes) {
        return teamLikes().stream()
                .map(LikeResponse::fromSentTeamLike)
                .collect(Collectors.toList());
    }
```

Reply:
LikeResponse에서 쓰는 메서드가 서로 다릅니다

Reply:
그렇네요! 좀 더 줄일 수 있을 것 같은데 함수형 인터페이스를 선언해서 줄이는 방법까지는 불필요할까요?
이 부분은 추후 좀 더 고민해보면 좋을 것 같아요.